backend/spec/features/admin/products/properties_spec.rb

```diff
@@ -128,14 +130,14 @@ def fill_in_property
 
     def delete_product_property
       accept_alert do
-        click_icon :trash
+        all('.delete-resource').last.click
```

Comment:
I think we can use `within_row(1) do` here instead.

Reply:
ok. i would change that in the next refresh.

Reply:
ahh this does not work. before delete_product_property can delete any line doesn't matter. But in that specific state of the test it would have to delete the second line! the other tests would have to delete the first line. So i would need to add a param for the line number. i decide to not use the function for my case!